Makefile

```diff
@@ -18,11 +18,11 @@ create_db: start
 	done
```

Comment:
The host network change required a change from `run` to `exec`, not entirely sure why....